src/base/tests/test_util.py

```diff
@@ -1060,8 +1060,8 @@ def test_rename_xmlid(self):
 
     def test_update_record_from_xml(self):
         # reset all fields on a <record>
-        xmlid = "base.res_partner_title_mister"
-        data_after = {"name": "Fortytwo", "shortcut": "42"}
+        xmlid = "base.res_partner_industry_A"
+        data_after = {"name": "New Agriculture", "full_name": "New A - AGRICULTURE, FORESTRY AND FISHING"}
```

Comment:
The idea here is to write something different from the standard.
```suggestion
        data_after = {"name": "42", "full_name": "Fortytwo"}
```